src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -190,7 +190,7 @@ private void handleExit() {
     }
 
     public PersonListPanel getPersonListPanel() {
```

Comment:
Are we gonna keep the method name as getPersonListPanel instead of changing to getTaskListPanel for now?